tests/Features/Laravel8/Application/database/factories/UserFactory.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Illuminate\Support\Str;
+use Laravel8\Models\User;
+
+/**
+ * @extends Factory<User>
```

Comment:
Will our users need this?

Reply:
Yes. I'll document it in release notes or upgrade guide.

Reply:
Wondering if we could infer this from the class name or from the `$model` property?

Reply:
No. As far as I know PHPStan does not support that. If you are extending a generic class, you have to have the `@extends`

Reply:
@canvural Let me give this a shot tonight if you don't mind.

Reply:
Yeah sure. But there is nothing you do I think. Unless you find a solution without making `Factory` class generic.

Reply:
You were right. There is way we make users know, that generics are missing their Laravel factories?

Reply:
PHPStan will complain if `@extends` is missing, in level 6 and above.